src/canopy_server/app.py

```diff
@@ -18,12 +18,7 @@
 from starlette.concurrency import run_in_threadpool
 from sse_starlette.sse import EventSourceResponse
 
-from fastapi import (
-    FastAPI,
-    HTTPException,
-    Body,
-    APIRouter
-)
+from fastapi import FastAPI, HTTPException, Body, APIRouter
```

Comment:
@MichaelAnckaert for this PR you can leave these changes as-is, but in the future please avoid running linters like `black` on files (or even lines) that weren't  changed in your PRs. This results in redundant conflicts with other PRs.